test/testfixture/make_functions.go

```diff
@@ -487,3 +488,26 @@ func makeLabels(fxt *TestFixture) error {
 	}
 	return nil
 }
+
+func makeTrackers(fxt *TestFixture) error {
+	if fxt.info[kindTrackers] == nil {
+		return nil
+	}
+	fxt.Trackers = make([]*tracker.Tracker, fxt.info[kindTrackers].numInstances)
+	trackerRepo := remoteworkitem.NewTrackerRepository(fxt.db)
+
+	for i := range fxt.Trackers {
+		fxt.Trackers[i] = &remoteworkitem.Tracker{
+			URL:  testsupport.CreateRandomValidTestName("tracker "),
```

Comment:
Use a real URL here.

Reply:
done as a part of this commit -> a585aef